notebooks/en/rag_with_unstructured_data.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>maybe we could divide this in smaller parts and put small explanations on what each code does. also here's a small ref for those who don't know bnb that we can add: <a href="https://huggingface.co/blog/merve/quantization" target="_blank">https://huggingface.co/blog/merve/quantization</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I skipped this to avoid making the notebook too long and because this is not the main part of the recipe. Instead, right in the beginning I give a link to the basic RAG notebook where all this is explained. </p>